careers/ladder.md

```diff
@@ -563,9 +575,66 @@ See
 [this document](https://www.notion.so/artsy/Artsy-Tech-Lead-Expanded-Responsibilities-Public-710ee0cd2fce47dcaa722ece80e365b2)
 for an expanded set of Tech Lead Responsibilities at Artsy.
 
+## Staff Engineer Archetypes
+
+For engineers who would like to continue growing on the IC track, archetypes can serve to clarify what it looks
+like for an IC to have an impact at the Staff Engineer level. These archetypes paint a picture of the different
+flavors of specialization that engineers can pursue at the Staff Engineer level once they have demonstrated their
+impact across the gamut of engineering activities at Artsy.
```

Comment:
> ...paint a picture of the different flavors...

(Mixed metaphor.)

More broadly: what problem are we trying to solve here? My assumption is that we are _not_ encouraging ICs to "specialize" in order to be recognized. Specialization tends to happen organically as engineers follow appealing problems and build expertise. Instead, I think we should be trying to communicate that we value [and recognize] various engineering inclinations, and they all have the capacity to achieve org-wide impact, even if the most visible examples or most recent promotion might reflect a profile unlike one's own.

I'd suggest making it even more explicit that. (_Bad version:_ "Engineers' impact on the Artsy organization can take different forms, all of which are valuable.")

Reply:
I see how this paragraph is telling engineers that they need to specialize in order to be recognized as Staff Engineers. 

To mix the metaphor even more, we want to invite engineers to a buffet and show that there are many ways to assemble a plate, all of which will fill you up.